<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefineColour_Dialog</class>
 <widget class="QDialog" name="DefineColour_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>589</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>589</width>
    <height>460</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Define colour</string>
  </property>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>412</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Index colour </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_51">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>364</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_52">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>338</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_53">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>312</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_54">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>338</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lightness</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_55">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>364</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saturation</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_56">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>312</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="_113">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>144</y>
     <width>90</width>
     <height>154</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QSlider" name="_106">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>364</y>
     <width>128</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="_105">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>338</y>
     <width>128</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="_104">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>312</y>
     <width>128</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="_112">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>362</y>
     <width>128</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="_111">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>336</y>
     <width>128</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="_110">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>312</y>
     <width>128</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="_100">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>18</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="_103">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>362</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="_102">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>336</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="_101">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>310</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QGraphicsView" name="_115">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>144</y>
     <width>90</width>
     <height>154</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>124</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New Colour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>488</x>
     <y>124</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Colour</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="_116">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>66</y>
     <width>183</width>
     <height>50</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="_120">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>10</y>
     <width>369</width>
     <height>289</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>16</height>
    </size>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="HLS">
    <attribute name="icon">
     <iconset resource="../RealCAD.qrc">
      <normaloff>:/images/HLS.png</normaloff>:/images/HLS.png</iconset>
    </attribute>
    <attribute name="title">
     <string notr="true"/>
    </attribute>
    <attribute name="toolTip">
     <string>HLS</string>
    </attribute>
    <widget class="QLabel" name="_121">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>360</width>
       <height>256</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="BW">
    <attribute name="icon">
     <iconset resource="../RealCAD.qrc">
      <normaloff>:/images/BandW.png</normaloff>:/images/BandW.png</iconset>
    </attribute>
    <attribute name="title">
     <string notr="true"/>
    </attribute>
    <attribute name="toolTip">
     <string>Grey Scale</string>
    </attribute>
    <widget class="QLabel" name="_122">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>360</width>
       <height>256</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Palette">
    <attribute name="icon">
     <iconset resource="../RealCAD.qrc">
      <normaloff>:/images/CPAL.png</normaloff>:/images/CPAL.png</iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <attribute name="toolTip">
     <string>Current Palette</string>
    </attribute>
    <widget class="QLabel" name="_123">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>2</y>
       <width>352</width>
       <height>256</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDoubleSpinBox" name="_107">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>358.999900000000025</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="_108">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>336</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>0.999900000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="_109">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>362</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>0.999900000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="_114">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>412</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>46</y>
     <width>97</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Screen Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>456</x>
     <y>412</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <zorder>_120</zorder>
  <zorder>_1</zorder>
  <zorder>label_42</zorder>
  <zorder>label_51</zorder>
  <zorder>label_52</zorder>
  <zorder>label_53</zorder>
  <zorder>label_54</zorder>
  <zorder>label_55</zorder>
  <zorder>label_56</zorder>
  <zorder>_113</zorder>
  <zorder>_106</zorder>
  <zorder>_105</zorder>
  <zorder>_104</zorder>
  <zorder>_112</zorder>
  <zorder>_111</zorder>
  <zorder>_110</zorder>
  <zorder>_100</zorder>
  <zorder>_103</zorder>
  <zorder>_102</zorder>
  <zorder>_101</zorder>
  <zorder>_115</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>_116</zorder>
  <zorder>_107</zorder>
  <zorder>_108</zorder>
  <zorder>_109</zorder>
  <zorder>_114</zorder>
  <zorder>label_3</zorder>
  <zorder>_2</zorder>
 </widget>
 <tabstops>
  <tabstop>_101</tabstop>
  <tabstop>_102</tabstop>
  <tabstop>_103</tabstop>
  <tabstop>_104</tabstop>
  <tabstop>_105</tabstop>
  <tabstop>_106</tabstop>
  <tabstop>_107</tabstop>
  <tabstop>_108</tabstop>
  <tabstop>_109</tabstop>
  <tabstop>_110</tabstop>
  <tabstop>_111</tabstop>
  <tabstop>_112</tabstop>
  <tabstop>_100</tabstop>
  <tabstop>_120</tabstop>
  <tabstop>_114</tabstop>
  <tabstop>_1</tabstop>
  <tabstop>_115</tabstop>
  <tabstop>_116</tabstop>
  <tabstop>_113</tabstop>
 </tabstops>
 <resources>
  <include location="../RealCAD.qrc"/>
 </resources>
 <connections/>
</ui>
